src/libstd/keyword_docs.rs

```diff
@@ -1030,22 +1030,39 @@ mod self_upper_keyword {}
 //
 /// A place that is valid for the duration of a program.
 ///
-/// A `static` item is similar to a [`const`] item in that it lives for the
-/// entire duration of the program and need to have its type explicited, with a
-/// `static` lifetime, outliving any other lifetime. Added to that, `static`
-/// items represent a precise memory location.
+/// A static item is a value which is valid for the entire duration of your
+/// program (a `'static` lifetime).
+///
+/// On the surface, `static` items seem very similar to [`const`]s: both contain
+/// a value, both require type annotations and both can only be initialized with
+/// constant functions and values. However, `static`s are notably different in
+/// that they represent a location in memory. That means that you can have
+/// references to `static` items and potentially even modify them, making them
+/// essentially global variables.
 ///
 /// Static items do not call [`drop`] at the end of the program.
 ///
 /// There are two types of `static` items: those declared in association with
 /// the [`mut`] keyword and those without.
 ///
+/// Items that are both static and owned cannot be moved:
```

Comment:
"Static items cannot be moved" is sufficient I think. It's true, too. (a) static items are mostly owned (they can only borrow from other statics) and (b) moving borrowed items is also not allowed; it just works because they are `Copy` and thus exhibit copy instead of move semantics.